css/styles.css

```diff
@@ -4,11 +4,64 @@
   --my-text-color: #172b4d;
   --my-green-color: #36b37e;
   --my-gray-color: #505f79;
+  --image-placeholder-color: #d0d9d4;
+  --image-placeholder-color1: #ebf0ee;
+  --my-project-boutton-color: #36b37f;
+  --my-competences-boxes-color: #c1c7d0;
 }
 
 html {
   background-color: none;
-  font-family: Inter, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-family: 'Inter', 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+/*
+buttons
+This is a requirement of linters
+Expected selector ".project-buttons a" to come before selector ".social li a"
+So I grouped all my buttons here
+*/
+.project-buttons a {
+  background-color: var(--my-project-boutton-color);
+  width: fit-content;
+  border: none;
+  color: white;
+  padding: 3% 3%;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 0% auto 1% auto;
+  border-radius: 4px;
+}
+
+.see-resume-button a {
+  padding: 10%;
+  margin: 3% auto 30% auto;
+  width: max-content;
+}
+
+.project-buttons:hover {
+  background-color: black;
+}
+
+.project-buttons:enabled {
+  background-color: white;
+}
+
+.see-resume-button:hover {
+  background-color: black;
+}
+
+.see-resume-button:enabled {
+  background-color: white;
 }
```

Comment:
- [ ] [OPTIONAL] Great efforts there, @diarisdiakite :clap: however, this `background` value of `black` is not conforming with the requirements on Figma. Kindly check, using the screenshot below as guide to get the correct values onFigma.
![image](https://user-images.githubusercontent.com/99715640/227749510-e14cb0b2-3c8b-43c3-987f-2f401f01f7d8.png)

Reply:
Well received, that will be corrected @AbayomiOlaoye